pkg/secrethub/me.go

```diff
@@ -0,0 +1,33 @@
+package secrethub
+
+import "github.com/secrethub/secrethub-go/internals/api"
+
+// MeService handles operations on the authenticated account.
+type MeService interface {
+	// Repos retrieves all repositories of the current user.
+	Repos() ([]*api.Repo, error)
+	// User retrieves the current users details.
+	User() (*api.User, error)
+}
+
+type meService struct {
+	repoService RepoService
+	userService UserService
+}
+
+func newMeService(repoService RepoService, userService UserService) MeService {
+	return meService{
+		repoService: repoService,
+		userService: userService,
+	}
+}
+
+// Repos retrieves all repositories of the current user.
+func (ms meService) Repos() ([]*api.Repo, error) {
```

Comment:
Should this be `Repos` or `ListRepos`? Using `Repos` kind of feels like you would expect a `RepoService` that's configured to work for your own repos or something?

Not sure though, also like the simplicity of a simple `Repos` call. Also, what else would we add? Not `Repo` (singular), maybe `Orgs`, but not `Org` etc. 

Anyway, your call both are good.

Reply:
I prefer `ListRepos`, because we use verbs for all other actions as well. Will change it.

Reply:
By the same logic, `User` will become `GetUser`.